src/kartograf/mapping_metrics/metric_mapping_rmsd.py

```diff
@@ -60,11 +69,17 @@ def get_rmsd_p(self, mapping:AtomMapping, accepted_distance_rmsd:float = 0.5, k_
         """
         rmsd = self.get_rmsd(mapping)
         beta = 1000 / (const.k * const.Avogadro * T)
-        V = k_hook * (rmsd-accepted_distance_rmsd)
-        p = np.exp(-beta * V) if(np.exp(-beta * V) < 1) else 1
-        return float(np.round(p,5))
+        V = k_hook * (rmsd - accepted_distance_rmsd)
+        p = np.exp(-beta * V) if (np.exp(-beta * V) < 1) else 1
+        return float(np.round(p, 5))
 
-    def get_score(self, mapping, accepted_distance_rmsd:float = 0, k_hook=10**0, T=298)->float:
+    def get_score(
+        self,
+        mapping,
+        accepted_distance_rmsd: float = 0,
+        k_hook: float = 1,
+        T: float = 298,
+    ) -> float:
         """
             returns a normalized value between 0 and 1, where 1.0 is the best and 0.0 is the worst score.
             The value is rounded to 2 digits.
```

Comment:
why are we aggressively rounding results?  This seems like something someone can do as a preference rather than something we enforce

Reply:
hm I removed it, I assumed certain units.